### ISSUE TITLE
MA0202 - false positive on comment

### ISSUE THREAD

**Issue by fforjanAVEVA** [meziantou/Meziantou.Analyzer#1255]
### Version of the Meziantou.Analyzer NuGet package

3.0.137

### Rule Identifier

MA0202

### Target Framework

net472

### C# Language version

C# 12

### Description

if the condition branch is around comments then the rule is wrongly triggered

### Reproduction Steps

Minimal code:

```c#
#if MY_CONDITION
        /// <param name="parameter">my parameter</param>
#else
        /// <param name="parameter">my parameter, cannot be null</param>
#endif
```

**Comment by meziantou** [comment]
I've updated the rule, so for comment-only branches, comments are compared textually to avoid false positives when comment content differs.
